boards/arm/fanstel_ev832x/doc/fanstel_ev832x.rst

```diff
@@ -0,0 +1,32 @@
+.. _fanstel_ev832x:
+
+Fanstel EV-BT832X
+#################
+
+Overview
+********
+
+BluNor BT832X is a powerful, highly flexible, Bluetooth Low Energy (BLE) using
```

Comment:
This doesn't read as an overview of the EV-BT832X development board. Maybe this:
```
The Fanstel EV-BT832X board is an evaluation board for the BluNor BT832X Bluetooth Low Energy module.
This board uses a Nordic nRF52832 SoC, with an ARM Cortex M4F MCU, 512KB flash,
64KB RAM, embedded 2.4GHz multi-protocol transceiver, power amplifier, and an
integrated PCB trace antenna, or an u.FL connector for external antenna.
```